core-bundle/src/Resources/contao/library/Contao/Template.php

```diff
@@ -375,6 +391,21 @@ public function previewRoute($strName, $arrParams=array())
 		return StringUtil::ampersand($strUrl);
 	}
 
+	/**
+	 * Return the preview content route
+	 *
+	 * @param mixed $objContent The content
+	 * @param array $arrParams  The route parameters
+	 *
+	 * @return string The route
+	 */
+	public function previewContentRoute($objContent, $arrParams = array())
```

Comment:
> Is it really necessary to have a separate method? Can we not enhance the existing `previewRoute()` method to handle `$objContent`?

Reply:
Changed in a972f8d81da4a457cfd3dcadcb32c77e1d3c88aa, though we're no longer generating a route now… I wonder if we should change that in the `AbstractController` as well? But the parent method in the `ControllerTrait` has `@final` on it?